Quran/AyahInfo.swift

```diff
@@ -7,18 +7,33 @@
 //
 
 import Foundation
+import CoreGraphics
 
 struct AyahInfo {
-    let pageNumber: Int
+    let page: Int
+    let line: Int
     let ayah: AyahNumber
     let position: Int
     let minX: Int
     let maxX: Int
     let minY: Int
     let maxY: Int
-    var rect: Rect {
-        get {
-            return Rect(x: minX, y: minY, width: maxX - minX, height: maxY - minY)
-        }
+
+    func rect(scale: CGFloat, xOffset: CGFloat, yOffset: CGFloat) -> CGRect {
+        return CGRect(x: CGFloat(minX) * scale + xOffset,
```

Comment:
According to the last comment, we need to remove the scaling from here also, since it has nothing to do with the Model. I suggest do the following:
1. Revert `var rect: Rect` back again.
2. Add the scaling function as an extension over CGRect in a new file named `CGRect+Extension.swift` in Classes -> Helper -> Extensions.